schemas/translation_key.yaml

```diff
@@ -13,6 +13,8 @@ translation_key:
       type: string
     plural:
       type: boolean
+    use_ordinal_rules:
```

Comment:
perhaps also add in the example below?

Reply:
yes I think it should be added there as well